build.gradle.kts

```diff
@@ -86,6 +87,15 @@ sonarqube {
     }
 }
 
+conventionalCommits {
+    scopes = subprojects.map { it.name }.toList()
```

Comment:
You might want to explicit the `it` variable.

Reply:
You're right, I explicited it